src/ralph/lib/transitions/models.py

```diff
@@ -424,7 +436,7 @@ def run_field_transition(
     for instance in instances:
         _post_transition_instance_processing(
             instance, transition, data, history_kwargs=history_kwargs,
-            user=kwargs['request'].user, attachments=attachments,
+            user=requester, attachments=attachments,
```

Comment:
maybe `user` param could be renamed to `requester` as well to keep it consistent?